Tests/InstrumentationTests/Tracing/TracedLock.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Baggage
+import Instrumentation
+
+final class TracedLock {
+    let name: String
+    let underlying: NSLock
+
+    var activeSpan: Span?
+
+    init(name: String) {
+        self.name = name
+        self.underlying = NSLock()
```

Comment:
Small nitpick. Can we name this `underlyingLock` instead? Just `underlying` feels a bit strange in Swift I'd say, although the _context_ makes it a bit clearer in this case.

Reply:
okey